python/convert.c

```diff
@@ -319,22 +319,23 @@ bool PyUpb_Array_IsEqual(const upb_array *arr1, const upb_array *arr2,
 
 bool PyUpb_Message_IsEqual(const upb_msg *msg1, const upb_msg *msg2,
                            const upb_msgdef *m) {
+  const upb_symtab* symtab = upb_filedef_symtab(upb_msgdef_file(m));
   size_t iter1 = UPB_MSG_BEGIN;
   size_t iter2 = UPB_MSG_BEGIN;
```

Comment:
can you move all of this below the fast path bail outs?

Reply:
Done.